config/impl-base/src/test/java/org/jboss/arquillian/config/impl/extension/PropertiesParserTestCase.java

```diff
@@ -468,6 +468,16 @@ public String get() {
         });
     }
 
+    @Test(expected = IllegalArgumentException.class)
+    public void shouldThrowExceptionIfDescriptorIsNull() {
+	new PropertiesParser().addProperties(null, System.getProperties());
```

Comment:
The indentation is completly off in all the changes FYI, please fix this next time. Thanks!

Reply:
Ah, even worse, these are tabs, but unfortunately, this project uses spaces...

Reply:
> Ah, even worse, these are tabs, but unfortunately, this project uses spaces...

Nothing unfortunate about that. It's what Charles Babbage would have wanted... :P